app/components/member/Edit/Memberships.js

```diff
@@ -27,6 +27,7 @@ class MemberEditMemberships extends React.PureComponent {
         ).isRequired
       }).isRequired
     }),
+    isSelf: PropTypes.bool.isRequired,
```

Comment:
Yours had it in the state, but in the end I moved it up the chain so the top-level element passes it to the children so they don’t all have to request `viewer` for that simple case. If they were to be used elsewhere they might need it again.

`isAdmin` remains state, and now lives in `member/Edit/role.js` (the only place it’s used)